agents/agents/guard/db/sql/base/base.go

```diff
@@ -33,6 +33,7 @@ func (s Store) SubmitterDB() submitterDB.Service {
 // GetAllModels gets all models to migrate.
 // see: https://medium.com/@SaifAbid/slice-interfaces-8c78f8b6345d for an explanation of why we can't do this at initialization time
 func GetAllModels() (allModels []interface{}) {
+	allModels = append(allModels, &Dispute{}, &AgentTree{}, &AgentRoot{})
```

Comment:
make this one line w/ 37

Reply:
Don't think there is a way to do this in one `append`, but can make it in one line by stringing together two `append`s. Going to keep separate for now.